packages/samples/tspconfig.yaml

```diff
@@ -1,2 +1,9 @@
 emit:
   - "@typespec/openapi3"
+
+linter:
```

Comment:
Would revert that for now, best practice is not a published package and this would give the wrong impression looking at samples